### PR TITLE
Windows episode

### PR DESCRIPTION
**Pull request by jayunit100**

### CODE REVIEW

File: episodes/144/README [hunk 1]
```diff
@@ -0,0 +1 @@
+ssh capv@10.176.37.216
```
Comment (jbeda) [inline episodes/144/README:1]: Do you need a README and a README.md? That might confuse what github shows.

File: episodes/144/README.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+- Recording date: 2021-01-28 
+
+<!--- Thumbnailed embed of the video, n8Xo_ghCIOSY is the video id from the youtube url --->
+
+<a href="https://www.youtube.com/watch?v=WnXnO97tsNo" target="_blank"><img src="https://i.ytimg.com/vi/WnXnO97tsNo/maxresdefault.jpg" border="10" /></a>
+
+## Table of Contents
+
+- 00:00:00 - Welcome to TGIK!
+- 00:00:00 - Week in Review
```
Comment (jbeda) [inline episodes/144/README.md:13]: I generally go through and find the main things in the timeline. At least call out when you start talking about the main topic. If you copy/paste this table into the youtube description then it creates chapter marks for folks.

**Review by jbeda — COMMENTED** [review]
Just some nits! Hopefully easy enough to get addressed.

**Comment by jbeda** [comment]
Also don't forget the DCO -- that is just putting a comment in the commit description: https://github.com/vmware-tanzu/tgik/blob/master/CONTRIBUTING.md

File: episodes/144/README.md (hunk 2 above)
Comment (jayunit100) [reply episodes/144/README.md:13]: ah yeah ! will do that
perfect

File: episodes/144/README (hunk 1 above)
Comment (jayunit100) [reply episodes/144/README]: fixed !

**Comment by jayunit100** [comment]
ok i added comments and did the signoff thingy !!!

**Pull request merged**